src/Compilers/CSharp/Test/Syntax/Parsing/CSharpParseOptionsTests.cs

```diff
@@ -21,6 +22,18 @@ private void TestProperty<T>(Func<CSharpParseOptions, T, CSharpParseOptions> fac
             Assert.Same(newOpt2, newOpt1);
         }
 
+        [Fact]
+        public void LanguageVersionFactsCurrentVersion()
```

Comment:
We have a number of tests that track changes to LangVersion. They are grouped together and all reference `LanguageVersionAdded_Canary`. Please add a check for `CurrentVersion` in one of them, or move this test and call `LanguageVersionAdded_Canary()`.

Reply:
OK. I copied VB.

Reply:
Actually, `LanguageVersionAdded_Canary` is in command line test, but this has nothing to do with command line. I think we are missing `LanguageVersionTests.cs` where we should put all our language version tests.

Reply:
Yes, a new test file would make sense. My main point is the tests should be together and all have the canary.

Reply:
The tests are together in VB. So I put the C# tests to the corresponding class. Refactoring the test classes seems beyond the scope of this PR. 

Why is canary needed? This test is written in such a way that doesn't need to be updated whenever we add a new version.

Reply:
I'll move the test into a separate file `LanguageVersionTests.cs/vb`

Reply:
The purpose of canary is to remind us to make appropriate adjustments when a version is added.
This means we should have a test that just asserts the version of `CurrentVersion`, with canary.

The test you added indeed doesn't need the canary. Thanks